src/main.rs

```diff
@@ -24,6 +24,7 @@ mod nameutil;
 mod parser;
 mod traits;
 mod version;
+mod versions;
```

Comment:
I'd prefer a useful module use the name `version` and the former `version` renamed to something else.